src/lib.rs

```diff
@@ -9,7 +9,6 @@
 // - Henry de Valence <hdevalence@hdevalence.ca>
 
 #![no_std]
-#![cfg_attr(feature = "nightly", feature(external_doc))]
 #![cfg_attr(feature = "nightly", doc(include = "../README.md"))]
```

Comment:
```suggestion
#![cfg_attr(feature = "nightly", doc = include_str!("../README.md"))]
```
Currently without this PR the `nightly` feature fails to compile.